Jenkinsfile

```diff
@@ -173,13 +173,32 @@ node {
         }
       }
 
-      stage("valgrind-memcheck-release") {
-        if (env.BRANCH_NAME == 'master' || full_ci) {
-          sh "mkdir ./clang-release-memcheck"
-          sh "valgrind --tool=memcheck --error-exitcode=1 --leak-check=full --gen-suppressions=all --suppressions=.valgrind-ignore.txt ./clang-release/hyriseTest clang-release-memcheck --gtest_filter=-NUMAMemoryResourceTest.BasicAllocate"
-          sh "valgrind --tool=memcheck --error-exitcode=1 --leak-check=full --gen-suppressions=all --suppressions=.valgrind-ignore.txt ./clang-release/hyriseSystemTest clang-release-memcheck"
-        } else {
-          Utils.markStageSkippedForConditional("memcheckClangRelease")
+      parallel memcheckReleaseTest: {
+        stage("memcheck-release-test") {
+          if (env.BRANCH_NAME == 'master' || full_ci) {
+            sh "mkdir ./clang-release-memcheck-test"
+            sh "valgrind --tool=memcheck --error-exitcode=1 --leak-check=full --gen-suppressions=all --suppressions=.valgrind-ignore.txt ./clang-release/hyriseTest clang-release-memcheck-test --gtest_filter=-NUMAMemoryResourceTest.BasicAllocate"
+          } else {
+            Utils.markStageSkippedForConditional("memcheckReleaseTest")
+          }
+        }
+      }, memcheckReleaseSystemTest1: {
+        stage("memcheck-release-systemtest1") {
+          if (env.BRANCH_NAME == 'master' || full_ci) {
+            sh "mkdir ./clang-release-memcheck-systemtest1"
+            sh "valgrind --tool=memcheck --error-exitcode=1 --leak-check=full --gen-suppressions=all --suppressions=.valgrind-ignore.txt ./clang-release/hyriseSystemTest clang-release-memcheck-systemtest1 --gtest_filter=TPCHTestInstances/TPCHTest.TPCHQueryTest/20"
```

Comment:
We're so gonna forget about removing this exclusion once TPCH20 runs fine :(

Reply:
It's not excluded, just executed somewhere else, right? Also, once that is the case, I'll become impatient again and balance the TPC-H queries more evenly :D

Reply:
kk